src/components/modals/settings-modal/settings-modal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Modal, Select } from "antd";
```

Comment:
We need to decide if we want to change from ant desing to other libraries 
Ofir and i found out that its effect performance